ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -523,6 +523,7 @@ objects:
       env:
       - name: TYPE
         value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NETWORK_MANIFEST
```

Comment:
You need to declare this as an optional template parameter or the value won’t get passed down to you.